spec/system/accessibility_page_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+RSpec.describe "Accessibility Page" do
+  before do
+    driven_by(:headless_chrome)
+    visit "/"
+    click_on "Reject additional cookies"
+    click_on "Hide cookie message"
+  end
+
+  it "displays an accessible page when link is clicked" do
```

Comment:
I wonder if this should be in `spec/system/accessibility_spec.rb`, which is more or less where all the AXE checks are?